src/main.cpp

```diff
@@ -1019,12 +1036,6 @@ static void MX_GPIO_Init(void) {
     HAL_GPIO_Init(GPIOC, &GPIO_InitStruct);
     HAL_GPIO_WritePin(GPIOC, ESP_RST_Pin, GPIO_PIN_RESET);
 #endif
-    /*Configure GPIO pins : FLASH_CSN_Pin */
-    GPIO_InitStruct.Pin = FLASH_CSN_Pin;
-    GPIO_InitStruct.Mode = GPIO_MODE_OUTPUT_PP;
-    GPIO_InitStruct.Pull = GPIO_NOPULL;
-    GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_LOW;
-    HAL_GPIO_Init(GPIOD, &GPIO_InitStruct);
```

Comment:
Initialization of pin removed, but pin definition

`#define FLASH_CSN_Pin               GPIO_PIN_7`
`#define FLASH_CSN_GPIO_Port         GPIOD`

was not removed, also 

```
/*Configure GPIO pin Output Level */
    HAL_GPIO_WritePin(GPIOD, FLASH_CSN_Pin, GPIO_PIN_RESET);
```
was not removed

Reply:
Fixed